src/main/java/com/prgrms/needit/domain/notification/entity/Notification.java

```diff
@@ -0,0 +1,77 @@
+package com.prgrms.needit.domain.notification.entity;
+
+import com.prgrms.needit.common.domain.entity.BaseEntity;
+import com.prgrms.needit.common.enums.UserType;
+import com.prgrms.needit.domain.notification.entity.enums.NotificationContentType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.util.Assert;
+
+@Getter
+@Table(name = "notification")
+@Entity
+@NoArgsConstructor
+public class Notification extends BaseEntity {
+
+	@Column(name = "notified_user_id", nullable = false)
+	private Long notifiedUserId;
```

Comment:
Notification 도메인 안에 있는 필드이니 
```suggestion
	private Long userId;
```
로 해도 좋을 것 같아요!

Reply:
넵!